### DISCUSSION TITLE
Allow to pass log config to the start-reload.sh for the uvicorn

### DISCUSSION THREAD

**Discussion by ievgennaida** [tiangolo/uvicorn-gunicorn-fastapi-docker#318] [Questions] [answered]
Allow to pass log config to the start-reload.sh for the uvicorn.

**Comment by tiangolo** [comment, marked as answer]
Now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker#-warning-you-probably-dont-need-this-docker-image

Using Uvicorn directly with its configs would probably be much simpler than dealing with both Uvicorn and Gunicorn.
